lupus_to_mqtt/Panel.py

```diff
@@ -204,7 +204,10 @@ def newSensor(self, data):
             return None
 
         if type == CONST.TYPE_DOOR_WINDOW:
-            return DoorWindowSensor(data, self)
+            if data.get('openClose') is not None:
+                return DoorWindowSensor(data, self)
+            else:
+                return None
```

Comment:
I think you can put "return AlarmSensor(data, self)" here. This way, the alarm status could be read by HomeAssistant without providing any additional functionality. Again, I unfortunately don't have a device to test this myself.